error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/ReactorTemplates.java

```diff
@@ -271,6 +271,32 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link Flux#onErrorComplete()} over more contrived alternatives */
```

Comment:
```suggestion
  /** Prefer {@link Flux#onErrorComplete()} over more contrived alternatives. */
```
Javadoc should end with a dot 😄.,